includes/class-bsf-custom-fonts-taxonomy.php

```diff
@@ -77,7 +77,7 @@ public function __construct() {
 		 *
 		 * @since 1.0.0
 		 */
-		public function create_custom_fonts_taxonomies() {
+		public static function create_custom_fonts_taxonomies() {
```

Comment:
@GauravKhupse We have called this function on object reference which should be converted to static call.
`add_action( 'init', array( $this, 'create_custom_fonts_taxonomies' ) );`

Reply:
Changes done!